README.md

```diff
@@ -15,8 +17,10 @@ Here, we attempt at solving the testing problem by proposing an extensive librar
 
 ## Installation
 
+stimulus is still under development, you can install it from test-pypi by running the following command:
+
 ```bash
-pip install stimulus-py
+pip install --index-url https://test.pypi.org/simple/ --extra-index-url https://pypi.org/simple stimulus-py==0.0.10
```

Comment:
sadly not to my knowledge, this is required because stimulus is on the test repository of pipy, however its dependencies are not, therefore we need to specify both when downloading the package and its dependencies.